apps/vh/admin-web/admin-web.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: vh-admin-web
+spec:
+  releaseName: vh-admin-web
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/vh/admin-web:prod-e23c232-20220117145616 # {"$imagepolicy": "flux-system:vh-admin-web"}
```

Comment:
are these specifc image refs intentional or just the same as what was in PIP? Could change em all to latest to avoid issues